.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, PDF: ${{ matrix.pdf }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltopdf: ${{ matrix.wkhtmltopdf }}"
```

Comment:
I wonder if it wouldn't be better to leave it as it was and hardcode the version of wkhtmltopdf in the proper step. IMO, this information is not important in the name of the build and in the matrix with supported versions 🤔

Reply:
The version cannot be hardcoded because of the cache. If the version is different, the cache will be invalidated. However, I can move the wkhtml version to env variables instead.

Reply:
On the other hand, this is the version of wkhtml we support and test so IMO having it in the matrix is ok. It's just a different info in the build name (exact version instead of `true` :stuck_out_tongue:)

Reply:
Ok, you've convinced me 👌🏻